README.md

```diff
@@ -22,49 +34,152 @@ You can do it manually following the instructions from [rust-lang.org](https://w
 
 Otherwise, CMake would not be able to compile Rust-related targets and will produce a warning about that during configuration.
 
-## Clone
+### CodeStyle
 
-To clone repository execute
-```
-git clone https://github.com/filecoin-project/cpp-filecoin.git
-cd cpp-filecoin
-git submodule update --init --recursive
-```
+We follow [CppCoreGuidelines](https://github.com/isocpp/CppCoreGuidelines).
+
+Please use clang-format 11.0.0 with provided [.clang-format](.clang-format) file to autoformat the code.
+
+## Maintenance
+
+Maintainers: @zuiris, @turuslan, @Elestrias, @ortyomka, @wer1st, @Alexey-N-Chernyshov
+
+Tickets: Can be opened in Github Issues.
+
+
+# Cpp-Filecoin install and setup guide
+## Fuhon node establishing
+There are 2 ways of getting executable version of filecoin node and miner. The First is to get executables by downloading, you can do it here: `https://link-tofiles`.The second is to build executables by yourself, see **Advanced building**. To begin running a node you will need `node` executable.
+
+1. To configure node you also need to make:
+* **configuration file**
+  example `config.cfg`:
+
+     ```
+     bootstrap=<enter your peer here>
+     drand-server=api.drand.sh
+     drand-pubkey=868f005eb8e6e4ca0a47c8a77ceaa5309a47978a7c71bc5cce96366b5d7a569937c529eeda66c7293784a9402801af31
+     drand-genesis-time=1595431050
+     drand-period=30
+     ```
+* **genesisfile** - this is a special pre-set file with the initial block, which will configure the initial state of your node, set network parameters like sector size, recall time, e.t.c and define miner-list with first network miner.
+
+you also can download that [snapshots](https://fil-chain-snapshots-fallback.s3.amazonaws.com/mainnet/minimal_finality_stateroots_latest.car "snapshote store") for a less time-assuming connection and synchronisation to the popular filecoin networks.
+More about snapshot management [here](https://docs.filecoin.io/get-started/lotus/chain/#restoring-a-custom-snapshot "snapshot documentation").
+2. After all the preparations you can launch a node:
+* to start executable`$~/.../PATH_TO_EXECUTABLE`
+*  also, you need to link working directory, which will be a storage for all node-associated files. It can be done with `--repo` flag
+*  One more important linking is configuration and genesis files, you can pass paths to them with `--config "CONFIG_FILE"` and `--genesis "GENESIS_FILE"` respectively.
```

Comment:
quotes (`"`) in `--config "CONFIG_FILE"` are misleading, because they have meaning in shell commands.

my previous comment was ambigous, because i wrote "CONFIG_FILE" instead of `CONFIG_FILE`.

Reply:
CONFIG_FILE is already a part of a command layout, so it can't lay in a  unique layer